dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractCrudController.java

```diff
@@ -165,6 +167,9 @@
     // OLD PATCH
     // --------------------------------------------------------------------------
 
+    @OpenApi.Ignore
```

Comment:
Just curious about the effect of ignoring and adding params annotations at the same time?

Reply:
Ignore wins, I first went through adding parameters, later decided to ignore this one. We could remove the added parameter annotations now.

Reply:
Got it.